src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -107,11 +108,11 @@ private void OnMigrationAssistantRequested(object sender, EventArgs e)
                 throw new NullReferenceException(nameof(e));
 
             var btn = routedEventArgs.OriginalSource as System.Windows.Controls.Button;
-            var parentWindow = Window.GetWindow(btn);
+            var parentWindow = Window.GetWindow(btn) as ScriptEditorWindow;
 
             var node = sender as PythonNode;
-            var viewModel = new PythonMigrationAssistantViewModel(node);
-            var assistantWindow = new VisualDifferenceViewer(viewModel, parentWindow);
+            var viewModel = new PythonMigrationAssistantViewModel(node, parentWindow);
+            var assistantWindow = new VisualDifferenceViewer(viewModel);
```

Comment:
Try setting the owner here to avoid UI references on ViewModel
```suggestion
            var assistantWindow = new VisualDifferenceViewer(viewModel)
            {
                Owner = parentWindow
            };
```